### PR TITLE
ENT-4492: cf-monitord now honors monitorfacility in body monitor control

### PR DESCRIPTION
**Pull request by craigcomstock**
Ticket: ENT-4492
Changelog: title

### CODE REVIEW

**Comment by craigcomstock** [comment]
@cf-bottom jenkins please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8541/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8541/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8541/

**Comment by craigcomstock** [comment]
arm_64 failures are a new known issues: https://tracker.mender.io/browse/ENT-9129 and https://tracker.mender.io/browse/ENT-9130

File: cf-monitord/cf-monitord.c [hunk 1]
```diff
@@ -298,17 +298,26 @@ static void KeepPromises(EvalContext *ctx, const Policy *policy)
...
             }
 
             if (strcmp(cp->lval, CFM_CONTROLBODY[MONITOR_CONTROL_FORGET_RATE].lval) == 0)
             {
                 sscanf(value, "%lf", &FORGETRATE);
                 Log(LOG_LEVEL_DEBUG, "forget rate %f", FORGETRATE);
+                continue;
+            }
+
+            if (strcmp(cp->lval, CFM_CONTROLBODY[MONITOR_CONTROL_MONITOR_FACILITY].lval) == 0)
```
Comment (larsewi) [inline cf-monitord/cf-monitord.c:317]:
```suggestion
            if (StringEqual(cp->lval, CFM_CONTROLBODY[MONITOR_CONTROL_MONITOR_FACILITY].lval))
```
Maybe use `StringEqual`?

**Review by larsewi — DISMISSED** [review]
One minor nitpick, otherwise LGTM! :rocket:

File: cf-monitord/cf-monitord.c (hunk 1 above)
Comment (craigcomstock) [reply cf-monitord/cf-monitord.c:317]: indeed

**Comment by craigcomstock** [comment]
@cf-bottom jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/8545/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/8545/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-8545/

**Review by larsewi — APPROVED** [review]
ACK

**Review by vpodzime — APPROVED** [review]
ACK

**Comment by craigcomstock** [comment]
cherry picked to 3.18.x here: https://github.com/cfengine/core/pull/5044

**Pull request merged**